lms/djangoapps/courseware/course_tools.py

```diff
@@ -73,3 +74,63 @@ def url(cls, course_key):
         """
         request = get_current_request()
         return verified_upgrade_deadline_link(request.user, course_id=course_key)
+
+
+class FinancialAssistanceTool(CourseTool):
+    """
+    The financial assistance tool.
+    """
+    @classmethod
+    def analytics_id(cls):
+        """
+        Returns an id to uniquely identify this tool in analytics events.
+        """
+        return 'edx.tool.financial_assistance'
```

Comment:
Let's make a note on this to check again with data team or at least find out how's the approach to notify about this new id

Reply:
thanks Juliana! Yes here's the open discussion:
https://openedx.atlassian.net/browse/BISUPPORT-1632

Reply:
Perfect!